### PR TITLE
Update Patroni

### PR DESCRIPTION
**Pull request by Sybrand**
- Update patroni to match latest from bcdevops.
- Update postgis to match version update from bcdevops.
- Added network security policy to allow builds.
- Update readme's.

### CODE REVIEW

File: openshift/patroni-postgis/README.md [hunk 1]
```diff
@@ -13,18 +13,25 @@ Copied from [BCDevOps](https://github.com/bcdevOps/platform-services/) which was
...
 # Tag the new imagestream (it won't be used until the pods get re-created):
-oc -n auzhsi-tools tag patroni:v10-latest patroni:v10
+oc -n e1e498-tools tag patroni:v11-latest patroni:v11
 ```
 
-Allow the production product to pull images from tools
+#### Other examples of building and tagging
+
+```bash
+# Build a patroni imagestream, overide the git branch:
```
Comment (andrea-williams) [inline openshift/patroni-postgis/README.md:30]:
```suggestion
# Build a patroni imagestream, override the git branch:
```

Comment (andrea-williams) [reply openshift/patroni-postgis/README.md:30]: I know that typos in comments are not at all important, I just gotta live up to my reputation ;)

File: openshift/patroni-postgis/docker/contrib/root/usr/share/scripts/patroni/health_check.sh [hunk 2]
```diff
@@ -2,6 +2,9 @@
 set -Eeu
 set -o pipefail
 
+# check for disk space. Fails if usage hits above 90%
+# df "${PATRONI_POSTGRESQL_DATA_DIR:-/home/postgres/pgdata}" --output=pcent | tail -n 1 | awk '{if ($1+0 > 90) exit 1; else exit 0;}'
```
Comment (andrea-williams) [inline openshift/patroni-postgis/docker/contrib/root/usr/share/scripts/patroni/health_check.sh:6]: why is this commented out?

Comment (Sybrand) [reply openshift/patroni-postgis/docker/contrib/root/usr/share/scripts/patroni/health_check.sh:8]: I've added an explanation (I was keeping this code in, as that's what the upstream version does here)

**Review by andrea-williams — APPROVED** [review]

**Pull request merged**